src/eos-crash-metrics.c

```diff
@@ -134,15 +135,25 @@ is_blacklisted_path (const char *path)
 int
 main (int argc, char **argv)
 {
-  const gchar *path = argv[1];
-  const gint16 signal = atoi (argv[2]);
-  const gint64 timestamp = atoll (argv[3]);
+  const gchar *path = NULL;
+  gint16 signal = 0;
+  gint64 timestamp = 0;
```

Comment:
Not sure I need to initialize here, as atoi and atoll do default to 0 if the input can not be converted.